db/migrate/20221003153000_add_ansible_callback_enabled_to_templates.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddAnsibleCallbackEnabledToTemplates < ActiveRecord::Migration[6.0]
+  def change
+    add_column :templates, :ansible_callback_enabled, :boolean, default: false
+    Template.where(name: 'Ansible Roles - Ansible Default').update_all(ansible_callback_enabled: true)
```

Comment:
In here we should also find the templates which are assigned to `ansible_run_host` feature and set the flag for them as well.

We could even go one step further and set the flag only for the templates with the feature to not introduce any surprising behavior.

Reply:
> You mean that we should give the option to enable the callback only for templates assigned to the ansible_run_host feature?

Yes, but only in this migration

> And also implement it in the UI?

No, users should be free to do whatever they please. It is just that the migration should honor whatever changes the users might have already made